infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -27,6 +27,20 @@ exports.tasks.push({
         'project:taskcluster:smoketest:abc/*' ],
     };
     assert.deepEqual(expandedRole.scopes, expectedScopes.scopes);
-    await auth.deleteRole(roleId);
+    const query = {};
+    const anHourAgo = Date.now() - (1000*60*60);
+    while (1) {
+      const res = await auth.listRoles2();
+      for(let i=0;i<res.roles.length;i++){
```

Comment:
The more "modern" way to do this in JS is either
```
res.roles.forEach(role => {...});
```
or
```
for (let role of roles) { ... }
```

Reply:
Yes, this approach would've been more clean and modern.